lib/features/home_screen/widgets/contact_icon_widget.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../../config/app_ui_config.dart';
+import '../../../theme/app_theme.dart';
+
+class ContactIconWidget extends StatelessWidget {
+  const ContactIconWidget({super.key, required this.url});
+
+  final String url;
+
+  @override
+  Widget build(BuildContext context) {
+    final Uri _url = Uri.parse('https://flutter.dev');
+
+    return GestureDetector(
+      onTap: () async {
+        if (!await launchUrl(_url)) {
```

Comment:
At toPWr @simon-the-shark  created handy extension for launching urls like this.


``` dart
class LaunchUrlUtil {
  static Future<bool> launch(String uriStr) async {
    final uri = Uri.parse(uriStr);
    if (await canLaunchUrl(uri)) {
      return await launchUrl(uri);
    }
    return false;
  }
}
```

Reply:
I think global util like this in `lib/utils/lauch_url_util.dart` may be handy to create it once and use everywhere in the project